bbc/simorgh
# WSTEAM1-679 - Adds NewsArticle type and author for Live page SEO

Resolves JIRA [WSTEAM1-679](https://jira.dev.bbc.co.uk/browse/WSTEAM1-679)

Overall changes
======
Adds NewsArticle type and author for Live page SEO

Code changes
======

- Changes type from CollectionPage, to NewsArticle
- Enables Author for LinkedDataContainer

Testing
======
1. Visit http://localhost:7081/pidgin/live/c7p765ynk9qt
2. Use the SEO extension to check values are present

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
